pkg/object/object.go

```diff
@@ -157,11 +205,22 @@ func (c *Container) validate() error {
 			return fmt.Errorf("failed to validate mount: %v", err)
 		}
 
+		var mountType *string
```

Comment:
`mountType` is really confusing name, when there is no type associated here but just names.